src/analyzer/presenters/html/HTMLPresenter.java

```diff
@@ -0,0 +1,32 @@
+package analyzer.presenters.html;
+
+import analyzer.controller.Presenter;
+import analyzer.interactor.Report;
+import analyzer.views.html.HTMLActivity;
+import com.sun.istack.internal.NotNull;
+
+/**
+ * Is used in Controller to show data in HTML format.
+ *
+ * @author Lizaveta Havina
+ */
+public class HTMLPresenter implements Presenter {
```

Comment:
presenter has to have a dependency on view and call its method in showReport()
What if Report changes from call to call ?
Is it a good idea to accept report as a parameter ?